src/clj/rems/css/styles.clj

```diff
@@ -395,7 +403,7 @@
             :margin-bottom (u/em 1)}]
    [(s/descendant :.logo :.img) {:height "100%"
                                  :background-color (util/get-theme-attribute :logo-bgcolor)
-                                 :background-image (resolve-image (util/get-theme-attribute :logo-name))
+                                 :background-image (get-logo-image context/*lang*)
```

Comment:
should there also be a fallback when the language-specific image isn't defined? or YAGNI?

Reply:
The fallback is there, but it's actually in the `get-logo-image` fn.